generators/app/templates/jsconfig.json

```diff
@@ -11,7 +11,6 @@
       "~screens/*": ["./src/app/screens/*"],
       "~config/*": ["./src/config/*"],
       "~constants/*": ["./src/constants/*"],
-      "~redux/*": ["./src/redux/*"],
```

Comment:
I don't think it is a discussion for now, but should we use `~` or `@` in Viajeros we had some problems using ~ as it assumed all of those roots were imported from node modules. In amex we used @ and we could avoid the `internal/external` package problem using the following eslint configuration.
```
'import/order': [
      'error', {
        'newlines-between': 'always',
        groups: ['builtin', 'external', ['unknown', 'internal'], 'parent', 'sibling', 'index']
      }
    ]
```

With this eslint can differenciate between @apollo and @components. I think it could be a good improvment, what do you think?

Reply:
I think I had issues with jest for some reason when I tried it to use @. Can you add a card in here https://trello.com/b/sZVeiFgw/departamento-de-frontend so we can do this separatedly?